### PR TITLE
Added script to auto-create a module structure #33 #36

### PR DESCRIPTION
**Pull request by Shylajhaa**
Directory structure creation is completed. 
Yet to learn about CMakesList.txt and work on it.

### CODE REVIEW

**Comment by fabien-zoccola** [comment]
I was a bit disturbed upon seeing that you are using a shell file to run the python script. Would there be any way to get rid of said shell file? I was thinking of Python 3 and the `argparse` library. What do you think about it?

File: README.md [hunk 1]
```diff
@@ -24,6 +29,7 @@ Main repository: **[ArkScript](https://github.com/ArkScript-lang/Ark)**
 * C++17
 * CMake >= 3.12
```
Comment (PierrePharel) [inline README.md:30]: I gess, must be CMake >= 3.8

**Comment by Shylajhaa** [comment]
> I was a bit disturbed upon seeing that you are using a shell file to run the python script. Would there be any way to get rid of said shell file? I was thinking of Python 3 and the `argparse` library. What do you think about it?

Yeah this makes more sense, made the necessary changes

File: README.md (hunk 1 above)
Comment (Shylajhaa) [reply README.md:30]: Is this something to be changed?

Comment (PierrePharel) [reply README.md:30]: yes, i guess

Comment (Shylajhaa) [reply README.md:30]: Right, will update with other comments

**Comment by Shylajhaa** [comment]
Completed CMakesList creation. Referred to few files in the repo to arrive at the template

File: shell/createmodules/create.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+	'''
+	jsonFile = os.getcwd() + os.path.sep + cs.MODULE_CREATION_SHELL_DIR + os.path.sep + cs.MODULE_STRUCTURE_FILE_NAME
+	f = open(jsonFile)
+	directoryStructure = json.load(f)
+	f.close()
+	
+	return directoryStructure
+
+if __name__ == "__main__":
+	# print(cs)
```
Comment (SuperFola) [inline shell/createmodules/create.py:134]: You might want to remove this before merging, otherwise this seems good to me! That's a great work 😌

Comment (Shylajhaa) [reply shell/createmodules/create.py]: Done

File: README.md (hunk 1 above)
Comment (Shylajhaa) [reply README.md]: Made this change in the current commit

**Comment by SuperFola** [comment]
Seems good to me, LGTM! Don't forget to sign the CLA @Shylajhaa

**Comment by Shylajhaa** [comment]
Hi,

So, the licence mentioned about representations - that my current employer authorizes me to contribute.
Can you detail on that? Why are employer authorizations necessary for open-source contributions?


Regards

Shylajhaa

________________________________

**Review by SuperFola — APPROVED** [review]

**Comment by Shylajhaa** [comment]
> Seems good to me, LGTM! Don't forget to sign the CLA @Shylajhaa

Signed the CLA, can be merged

**Pull request merged**